sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DisposableTypesNeedFinalizers.cs

```diff
@@ -37,21 +37,8 @@ public class Foo_Compliant : IDisposable // Error [CS0535]
             private IntPtr myResource;
             private bool disposed = false;
 
-            protected virtual void Dispose(bool disposing)
```

Comment:
I removed this because the rule currently works with an empty `Dispose()`, so no need to add complex logic in the test